src/styles/card.scss

```diff
@@ -0,0 +1,106 @@
+.card {
+  width: $card-width;
```

Comment:
do not set width to bem block use mix instead, u already have additional classes for that